packages/pluginutils/test/dataToEsm.ts

```diff
@@ -2,6 +2,13 @@ import test from 'ava';
 
 import { dataToEsm } from '../';
 
+test('support bigint and symbol', (t) => {
```

Comment:
please split these into separate tests. (that may seem pedantic, but this PR essentially adds two features in support for two different entities)